cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -213,7 +213,7 @@
 
     {# TODO: Remove Else conditional when this is fully moved to wagtail. #}
     {# Check and include page-level JavaScript. #}
-    {% if page and page.media %}
+    {% if page %}
```

Comment:
If there is a page but no page.media, no js will be included - is this intended or is that a situation we no longer anticipate?